training/validate_myanmar.cpp

```diff
@@ -135,6 +135,7 @@ bool ValidateMyanmar::ConsumeOptionalSignsIfPresent() {
 /* static */
 bool ValidateMyanmar::IsMyanmarLetter(char32 ch) {
   return (0x1000 <= ch && ch <= 0x102a) || ch == 0x103f ||
+         (0x104c <= ch && ch <= 0x104f) ||
```

Comment:
Is it better to use a separate condition like you have done it, or could it be combined with the next line? The new condition ends at 0x104f and the next condition starts at 0x1050.